.github/workflows/build-components.yml

```diff
@@ -21,39 +21,9 @@ jobs:
 
       - name: Install gcds-components
```

Comment:
You can remove the `install gcds-components` step since the previous step(`install repo`) also runs `npm install`.

Reply:
Oh yes great catch!